src/components/select-project/select-project.controller.ts

```diff
@@ -113,6 +113,12 @@ export class SelectProjectController implements angular.IController {
     }
   }
 
+  public onOpenClose (isOpen: boolean) {
+    if (isOpen) {
+      this.$scope.$emit('select-project-openned');
```

Comment:
This feels more like it should be an optional callback passed into the component instead of `$scope.$emit` (similar to `onProjectSelected`).

Curious how it's used. To select the radio when the dropdown is opened?

Reply:
Yes, to select the radio when the dropdown is opened.  I can switch to a callback.